client/src/components/ResponseHandler.tsx

```diff
@@ -15,7 +15,7 @@ function ResponseHandler() {
         const data = await response.json();
         console.log(data);
         setConversation((prev) => [...prev, data.message]);
-        loaded = true;
+        setLoaded(true);
```

Comment:
This flag seems redundant to me. `useEffect` is only called once, right? So this code that goes and fetches from `/start` should only be called once, and there's nothing else that triggers this code path. So we don't need to check for `!loaded` on line 14 and that's the only place `loaded` is used as far as I see? Lmk if I'm missing something.